README.md

```diff
@@ -67,55 +67,55 @@ mv /tmp/INAV\ Configurator /opt/inav/inav-configurator
 sudo mkdir /opt/inav/inav-configurator/icon
 sudo cp /opt/inav/inav-configurator/images/inav_icon_128.png /opt/inav/inav-configurator/icon
 ```
-9. As a one off, move the desktop file into the applications directory 
+9. As a one-off, move the desktop file into the applications directory 
 ```
 sudo mv inav-configurator.desktop /usr/share/applications/
 ```
 10. Make the following files executable:
    * inav-configurator `chmod +x /opt/inav/inav-configurator/inav-configurator`
    * (5.0.0+) chrome_crashpad_handler `chmod +x /opt/inav/inav-configurator/chrome_crashpad_handler`
-11. Run INAV Configurator app from unpacked folder `/opt/inav/inav-configurator/inav-configurator`
+11. Run the INAV Configurator app from the unpacked folder `/opt/inav/inav-configurator/inav-configurator`
 
 #### Notes
 
-On some Linux distros, you may be missing `libatomic`, a `NW.JS` (specially `libnode.so`) dependency. If so, please install `libatomic` using your distro's package manager, e.g:
+On some Linux distros, you may be missing `libatomic`, an `NW.JS` (especially `libnode.so`) dependency. If so, please install `libatomic` using your distro's package manager, e.g:
```

Comment:
I think this would be better as:

missing `libatomic` and/or `NW.JS` (especially `libnode.so`) dependencies.